x-pack/plugins/session_view/public/components/session_view_detail_panel/index.tsx

```diff
@@ -56,11 +56,17 @@ export const SessionViewDetailPanel = ({
         content: <DetailPanelProcessTab selectedProcess={selectedProcess} />,
       },
       {
-        id: 'host',
-        name: i18n.translate('xpack.sessionView.detailsPanel.host', {
-          defaultMessage: 'Host',
+        id: 'metadata',
+        name: i18n.translate('xpack.sessionView.detailsPanel.metadata', {
+          defaultMessage: 'Metadata',
         }),
-        content: <DetailPanelHostTab processHost={selectedProcess?.events[0]?.host} />,
+        content: (
+          <DetailPanelHostTab
```

Comment:
We should probably rename this component to `DetailPanelMetadataTab` as well as its folder to `detail_panel_metadata_tab`